### PR TITLE
Add $BADKID asset

### PR DESCRIPTION
**Pull request by Dizz02**
### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - The `description` and/or `extended_description` of the asset in the Chain Registry is informative.
- [x] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_unlisted` defaults to `true` (until the respesentation and transferring of the new asset has been validated)
   - Note that it is recommended to include an X (fka, Twitter) profile URL with each asset.

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [x] Ready -- A liquidity pool has been created. The pool ID is: __1466__
- [ ] Soon -- A pool will be created. (See: [Pool Setup Guilde](https://docs.osmosis.zone/overview/integrate/pool-setup).)
  - [ ] (optional) A preview of the Osmosis Zone app with the new asset added is requested for creating the pool. (Supercharged Liquidity pools cannot be created via Osmosis Zone app)
- [ ] StreamSwap -- The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds.

### CODE REVIEW

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -13566,6 +13566,31 @@
         "osmosis_unlisted",
         "osmosis-unlisted"
       ]
+    },
+    {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:13570]: Please do not edit this osmosis-1.assetlist.json file directly.

Comment (Dizz02) [reply osmosis-1/osmosis-1.assetlist.json]: Done sorry

**Comment by JeremyParish69** [comment]
Please see the checklist section on Liquidity. What is the plan? or what is the pool id?

**Comment by Dizz02** [comment]
Pool created 27C112F008A1F6202A9A3267D1C96C957009040E7F42571972C4818C19869209 (1466) Thank you

**Comment by Dizz02** [comment]
Thank you for the assist Jeremy ;) I can see the pool on the main branch but cannot see the denom and can only trade osmo. Is the merge still in process?

**Comment by JeremyParish69** [comment]
> Thank you for the assist Jeremy ;) I can see the pool on the main branch but cannot see the denom and can only trade osmo. Is the merge still in process?

Can be seen now here:
https://app.osmosis.zone/assets?show_unlisted_assets=true

**Comment by JeremyParish69** [comment]
> Pool created 27C112F008A1F6202A9A3267D1C96C957009040E7F42571972C4818C19869209 (1466) Thank you

Uh 30% swap fee?

**Comment by Dizz02** [comment]
My bad. I forgot a zero :/ here's the replacement one created 1470. Can we completely remove the previous one from the frontend and they asked me if it's possible to list it right away "not_listed: false)?

**Comment by JeremyParish69** [comment]
Ok, set as 'listed' now. Should update on the frontend in ~10 mins

**Comment by Dizz02** [comment]
Thank you :)

**Pull request merged**